index.template.html

```diff
@@ -3282,6 +3293,73 @@ <h3>Semantic Annotations</h3>
         of the lamp.
     </p>
 
+</section>
+<section id="semantic-annotations-example-geoloc">
+    <h3>Example III: Geolocation Annotations</h3>
+
+    <p>
+    For some use cases like in building, agraculture, or smart city location based data is required. This
+    information can be provided in the Thing Description in different ways and can be relyed on
+    different kind of location ontologies depending or purpose (e.g., indoor, outdoor).  Sample vocabulary 
+    can be used such as [[WGS84]] or [[schema.org]]. Also see [[sdw-bp]].
+    </p>
+    
+    <p>
+        The TD snippet below uses <code>lat</code> and <code>long</code> from the [[WGS84]] ontology to 
+        provide latitude and longitude metadata at Thing's top level. 
+    </p>
+
+    <pre class="example" id="saref-state-annotation-example">
+{
+    "@context": [
+        "https://www.w3.org/2019/wot/td/v1",
+        {
+            "wgs84": "http://www.w3.org/2003/01/geo/wgs84_pos#"        
+        }
+    ],
+    "@type": "Thing",
+    "wgs84:lat" : "26.58",
+    "wgs84:long" : "297.83",
+    ...
+    "properties": {
+        ...
+    
+}
+</pre>
+
+
+<p>
+        In some use cases geo based metadata have to be provided at the interaction level, e.g., 
+        if the Thing Descriptions reflect a virtuel Thing since it combines multiple Things.
+        Geo laction based data can be provided at he interaction level as shown in the next sample:  
+</p>
+
+<pre class="example" id="saref-state-annotation-example">
+{
+    "@context": [
+        "https://www.w3.org/2019/wot/td/v1",
+        {
+            "iot": "http://schema.org#"
+        }
+    ],
+    ...
+    "properties": {
+        "status": {
```

Comment:
I guess the example intends to show how e.g. to interpret

```
{
  "long": 40.75,
  "lat": 73.98,
  "height": 15
}
```

as

```
{
  "@context": "http://schema.org/",
  "schema:longitude": 40.75,
  "schema:latitude": 73.98,
  "schema:elevation": 15
}
```
.

I provide an example for this in the [JSON Schema in RDF](https://www.w3.org/2019/wot/json-schema#defining-a-json-ld-context-for-data-instances) vocabulary, which simply consists in linking to a JSON-LD context from the TD, as follows

```
{
  "properties": {
    "position": {
      "jsonld:context": {
        "schema": "http://schema.org/"
        "long": "schema:longitude",
        "lat": "schema:latitude",
        "height": "schema:elevation"
      },
      "type": "object",
      "properties": {
        "long": { "type": "number" },
        "lat": { "type": "number" },
        "height": { "type": "number" }
      }
    }
  }
}
```

The value for `jsonld:context` is an _inline_ JSON-LD context that provides a mapping to the keys defined in the JSON schema.

Reply:
Would it be also possible to use this example?

{
  "properties": {
    "position": {
      "type": "object",
      "properties": {
        "@type" : "schema:GeoCoordinates",
        "longitude": { "type": "number" },
        "latitude": { "type": "number" },
        "elevation": { "type": "number" }
      }
    }
  }
}

Reply:
as a further example, yes. I would keep the one with `jsonld:context`, though, because it corresponds to another requirement.

Reply:
Lets keep both approaches to show the different kind of options how semantic based annotations can be applied.

Reply:
Shouldn't it be:
```js
{
  "properties": {
    "position": {
      "type": "object",
       "@type" : "schema:GeoCoordinates",
      "properties": {
        "longitude": { "type": "number" },
        "latitude": { "type": "number" },
        "elevation": { "type": "number" }
      }
    }
  }
}
```
?

Reply:
yes, my bad.

Reply:
thanks for the hint. I updated the example.